ui/src/shared/containers/SetOrg.tsx

```diff
@@ -59,17 +91,100 @@ const SetOrg: FC<Props> = ({
     updateReportingContext({orgID: null})
 
     if (!orgs.length) {
-      router.push(`/no-orgs`)
+      history.push(`/no-orgs`)
       return
     }
 
     // else default to first org
-    router.push(`/orgs/${orgs[0].id}`)
-  }, [orgID])
+    history.push(`/orgs/${orgs[0].id}`)
+  }, [orgID, orgs.length])
+
+  const orgPath = '/orgs/:orgID'
 
   return (
     <SpinnerContainer loading={loading} spinnerComponent={<TechnoSpinner />}>
-      {children}
+      <Switch>
+        {/* Alerting */}
+        <Route path={`${orgPath}/alerting`} component={AlertingIndex} />
+        <Route
+          path={`${orgPath}/alert-history`}
+          component={AlertHistoryIndex}
+        />
+        <Route path={`${orgPath}/checks/:checkID`} component={CheckHistory} />
+
+        {/* Tasks */}
+        <Route path={`${orgPath}/tasks/:id/runs`} component={TaskRunsPage} />
+        <Route path={`${orgPath}/tasks/:id/edit`} component={TaskEditPage} />
+        <Route path={`${orgPath}/tasks/new`} component={TaskPage} />
+        <Route path={`${orgPath}/tasks`} component={TasksPage} />
+
+        {/* Data Explorer */}
+        <Route path={`${orgPath}/data-explorer`} component={DataExplorerPage} />
+
+        {/* Dashboards */}
+        <Route
+          path={`${orgPath}/dashboards-list`}
```

Comment:
should probably run a change like this past someone from product. people still like to use bookmarks and this will break the experience for those users

Reply:
anyone that has a bookmark to that will be routed to the updated route by: https://github.com/influxdata/influxdb/pull/18854/files#diff-5789c3a5afbf8bd08396173c9c29417d.  User experience should not change.